pages/common/pio-settings.md

```diff
@@ -0,0 +1,20 @@
+# pio settings
+
+> View and modify PlatformIO settings.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_settings.html>.
+
+- List all PlatformIO settings:
```

Comment:
I'm not even sure if this is actually what it does, but...
```suggestion
- List the names and values of all PlatformIO settings:
```

Reply:
Exactly that, but it also prints a short description for each

Reply:
I just edited the page while you edited your suggestions. Not exactly the same, but hopefully close enough.